crates/git_cmd/src/lib.rs

```diff
@@ -93,6 +93,11 @@ impl Repo {
         Ok(())
     }
 
+    pub fn fetch(&self, obj: &str) -> anyhow::Result<()> {
+        self.git(&["fetch", "origin", obj])?;
```

Comment:
The thing with "origin" is, that it would work on ci for sure but locally not necessarily.

I for instance use always just "o" as the upstream name. No "origin" present in my case. 
But this can be determined from git.

Reply:
Right now the rest of the code assumes "origin". I opened an issue for this: https://github.com/MarcoIeni/release-plz/issues/608
So I will merge this as is :)